db/migrate/20150906173134_create_teams.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTeams < ActiveRecord::Migration
+  def change
+    create_table :teams do |t|
+      t.belongs_to :account
+      t.belongs_to :team
```

Comment:
hm.. I was looking at this, does it not apply here?:
"By convention, Rails assumes that the column used to hold the foreign key on this model is the name of the association with the suffix _id added."
http://guides.rubyonrails.org/association_basics.html